### PR TITLE
HTML/CSS & JavaScript capstone project - Conference page

### PR DESCRIPTION
**Pull request by ehma90**
### Conference page
**On this webpage, I:**

- Add index.html file.
- Add a mobile.css file that styles the mobile view page.
- Add a desktop.css file that styles the desktop view page.
- Add a data.js file that holds the information on the 'Feature Speaker'.
- Add a main.js file that has some javascript function that loads: mobile menu and 'Feature Speaker' function.

Here is a link to the [video](https://www.loom.com/share/173d9e900e3d498eb3617efbc59a2ffa) on loom

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            <p class="position">${item.position}</p>
+            <div class="dash"></div>
+            <p class="experience">${item.experience}</p>
+        </div>
+    </div>
+    `;
+  inputSpeaker += element;
+  element = '';
+  return item;
+});
```
Comment (sja-thedude) [inline main.js]: - [x] Kindly make sure to add checks background image for all the images as per shown in the attached loom video.
- [x] Kindly make sure that the entire speaker section follows the width, height and spacing as per shown in the attached loom video.
- [x] Kindly make sure that you implement the more button as shown in the loom video attached.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+            <ul class="navlist">
+                <li><i class="fa fa-times-circle-o close" aria-hidden="true"></i></li>
+                <li><a href="index.html" class="nav-item">About</a></li>
+                <li><a href="#" class="nav-item">Program</a></li>
+                <li><a href="#" class="nav-item">Join</a></li>
+                <li><a href="#" class="nav-item">Sponsor</a></li>
+                <li><a href="#" class="nav-item">News</a></li>
+                <li><a href="#" class="nav-item tech">Tech Campaign</a></li>
+            </ul>
```
Comment (sja-thedude) [inline about.html:38]: - [x] Kindly make sure to add the home page option to the menu for the mobile version.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            </strong>
+
+            <ul class="navlist">
+                <li><i class="fa fa-times-circle-o close" aria-hidden="true"></i></li>
+                <li><a href="index.html" class="nav-item">About</a></li>
+                <li><a href="#" class="nav-item">Program</a></li>
+                <li><a href="#" class="nav-item">Join</a></li>
+                <li><a href="#" class="nav-item">Sponsor</a></li>
+                <li><a href="#" class="nav-item">News</a></li>
+                <li><a href="#" class="nav-item tech">Tech Campaign</a></li>
```
Comment (sja-thedude) [inline about.html:37]: - [x] Kindly make sure to fix the button design as per shown in the attached loom video.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                <p>Create your own project using the available technologies we have. </p>
+            </div>
+            <div class="program">
+                <i class="fa fa-trophy" aria-hidden="true"></i>
+                <h3>Awards</h3>
+                <p>See this year's award winners and awesome packages we have for them </p>
+            </div>
+            <div class="program">
+                <i class="fa fa-handshake-o" aria-hidden="true"></i>
+                <h3>Partnership</h3>
```
Comment (sja-thedude) [inline index.html:81]: - [x] Kindly make sure to fix this section for all screen sizes as per shown in the attached loom video.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+            <li><i class="fa fa-facebook  social" aria-hidden="true"></i></li>
+            <li><i class="fa fa-twitter  social" aria-hidden="true"></i></li>
+            <li><a href="#" class=" social text">English</a></li>
+            <li><a href="#" class=" social text">My Page</a></li>
+            <li><a href="#" class=" social text">Logout</a></li>
+        </ul>
+
+        <nav id="navbar" class="container-fluid ">
+            <strong class="about-logo nav-logo">
+                <a href="index.html"><i class="fa fa-snowflake-o" aria-hidden="true"></i><span>Ehma</span>Tech</a>
```
Comment (sja-thedude) [inline index.html:27]: - [x] Kindly make sure to fix and display the logo correctly for screen size less than the mobile version as per shown in the attached loom video.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+            <img src="imgs/partner2.png" alt="Toptal" class="toptal" width="150" height="80">
+        </div>
+    </section>
+
+    <footer id="footer">
+        <strong class="about-logo logo footer-logo">
+            <i class="fa fa-snowflake-o flake" aria-hidden="true"></i><span>Ehma</span>Tech
+        </strong>
+
+        <p class="footer-text"> 2017 Lorem ipsum dolor sit, amet dolor sit,amet
```
Comment (sja-thedude) [inline about.html]: - [x] Kindly make sure to remove the Partners section for `about desktop version`.
- [x] Make sure that the footer section has a black background for `about section` as per the attached loom video.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ehma90, This is @sja-thedude your code reviewer.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Git-flow
✔️ All Linters Passing

## Required 
- Kindly add the video presentation in the pull request with:-
- [x] Record a video presenting the features of the project.
- [x] Maximum length of the video is 5 minutes.
- [x] The video contains a clear and objective description of the project in professional English. 
- [x] The topic and purpose of the project are made clear.
- [x] The video contains a demo of the project, showing the pages in different screen sizes.
- [x] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
- [x] Kindly make all the changes below according to [this](https://www.loom.com/share/c7157afe9a7548c9b228a1db9f83c6c8) loom video and _Check the comments under the review._
- [x] Kindly make sure to display all the contents according to the correct width, height, size as per shown in the video attached.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js (hunk 1 above)
Comment (ehma90) [reply main.js]: - I have implemented changes to the background image.

The more button is optional as it isn't what is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) for this project

File: about.html (hunk 2 above)
Comment (ehma90) [reply about.html:38]: This is not required according to this [project](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) requirement.

File: about.html (hunk 3 above)
Comment (ehma90) [reply about.html:37]: We were asked to create two different screens for this [project:](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)
- Mobile: up to 768px wide.
- Desktop: 768px or wider.

File: index.html (hunk 4 above)
Comment (ehma90) [reply index.html:81]: I have implemented these changes.

File: index.html (hunk 5 above)
Comment (ehma90) [reply index.html:27]: We were asked to create two different for this [project:](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)

Mobile: up to 768px wide.
Desktop: 768px or wider.

File: about.html (hunk 6 above)
Comment (ehma90) [reply about.html]: I have implemented these changes.

**Review by isaacpitwa — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @ehma90  👋🏻 
Great job implementing the changes requested by the previous reviewer.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**